Assets/Scripts/Minion/States/MinionAttackStateSO.cs

```diff
@@ -68,7 +68,11 @@ private IEnumerator AttackCoroutine()
             float chargeDuration = ChargeLength / ChargeSpeed;
             startTime = Time.time;
             agentTransform.GetComponent<HealthPoints>().SetCanTakeDamage(false);
-            agentTransform.GetComponent<MeshRenderer>().material = chargeMat;
+            foreach (MeshRenderer meshRenderer in agentTransform.GetComponentsInChildren<MeshRenderer>())
+            {
+                meshRenderer.material = chargeMat;
+            }
```

Comment:
Check if there is another way to do this.


If not maybe we should make a script for this